test/stdlib/CreateObject.test.ts

```diff
@@ -0,0 +1,16 @@
+const { AssociativeArray, BrsObjects, CreateObject, BrsInvalid } = require("../../lib/brsTypes");
+const BrsError = require("../../lib/Error");
+
+describe("CreateObject", () => {
+    beforeEach(() => BrsError.reset());
```

Comment:
Is this needed if we're not checking errors?  I honestly don't know as other tests don't have it.

Reply:
You're right, it's not strictly necessary.  Doesn't hurt anything I suppose but it also doesn't add anything.

Reply:
Yep, copy 🍝 fail. Removed these.